setup.py

```diff
@@ -29,6 +23,8 @@
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.6",
```

Comment:
2.6 is not being tested in tox and is no longer supported by python itself. Do you still want to keep it here?